CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -322,7 +332,7 @@ public function providerFormRule($values) {
         }
       }
     }
-    $options = self::_getServieOptions();
+    $options = self::_getServiceOptions();
```

Comment:
typo in the function name and doing a grep showed this was the only place it was used

Reply:
I've noticed this a long time ago but was too scared to change. Good fix!